fastlane/lib/fastlane/actions/app_store_build_number.rb

```diff
@@ -21,7 +21,7 @@ def self.run(params)
       def self.get_build_number(params)
         UI.message("Login to App Store Connect (#{params[:username]})")
         Spaceship::Tunes.login(params[:username])
-        Spaceship::Tunes.select_team
+        Spaceship::Tunes.select_team(team_id: params[:team_id])
```

Comment:
Can we also add `team_name` in here as well?
```suggestion
        Spaceship::Tunes.select_team(team_id: params[:team_id], team_name: params[:team_name])
```

Reply:
Updated with both requested changes.